extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -151,28 +153,29 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					continue;
 				}
 
-				const newConnectionStringValue = await vscode.window.showInputBox(
-					{
-						title: constants.enterConnectionString,
-						ignoreFocusOut: true,
-						value: 'Server=localhost;Initial Catalog={db_name};User ID=sa;Password={your_password};Persist Security Info=False',
-						validateInput: input => input ? undefined : constants.valueMustNotBeEmpty
-					}
-				) ?? '';
-
-				if (!newConnectionStringValue) {
-					// go back to select setting quickpick if user escapes from inputting the value in case they changed their mind
-					continue;
+				// get connection string for sql server
+				let connectionString: string = '';
+				const connectionInfo = await vscodeMssqlApi.promptForConnection(true);
+				if (!connectionInfo) {
+					// User cancelled
+					return;
 				}
-
+				const connectionUri = await vscodeMssqlApi.connect(connectionInfo);
```

Comment:
This should probably be in the try/catch too so the user can select another connection if this one fails to connect. 

If you don't already I would also suggest that you make an issue tracking the removal of this connect call - we shouldn't need to require the user have a valid connection to populate the connection string. (and especially shouldn't require them actually connecting to said server to do it)